.github/workflows/compose_test.yaml

```diff
@@ -54,12 +54,13 @@ jobs:
     strategy:
       matrix:
         BE_VERSION: [v3, v4]
-        JOBS_AND_ELASTIC_ENABLED: ['', true]
+        FEATURE_ENABLED: ['', true]
```

Comment:
It might be good to test the combination of features (or maybe it's too much?). So the matrix could test V3 with jobs and LDAP; V3 with jobs; V3 with LDAP (this means adding a separate LDAP_ENABLED key to the matrix). jobs_and_elastic were merged because they have redundancy once BE_VERSION is set. 

It might be an overkill though

Reply:
I thought about it too but wasn't sure it was needed

Reply:
It might be useful I think, so we test the multiple merging rules work together. In another pr we can see if it's doable to cache layers in the ci